engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -163,13 +164,15 @@ def set_next_token(self,
         self.step_token_number = len(
             next_token.id) if next_token.id[0] != -1 else -1
         details = {}
+        # making detailed information captured for each token generation
+        if self.token_cache is not None:
+            details["finish_reason"] = finish_reason
+            details["tokens"] = self.token_cache
+            details["generated_tokens"] = len(self.token_cache)
+            details["input_text"] = self.input_text
```

Comment:
Can we send the entire payload instead of just input_text. We need to access other input parameters like maxtokens to calculate usage metrics along with input text